starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -92,7 +92,7 @@ async def test_throws_on_prepared_call_without_max_fee(map_contract):
     prepared_call = map_contract.functions["put"].prepare(key, value)
 
     with pytest.raises(
-        ValueError, match="Max_fee must be specified when invoking a transaction"
+        ValueError, match=r".*[mM]ax_fee must be specified when invoking a transaction"
```

Comment:
If #616 get's merged before this PR, these can be replaced. This is a temporary fix.

Reply:
It is merged 👀

Reply:
Merged development, these should be replaced now.